hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -266,7 +266,7 @@ public static void loadInstants(
           .filter(fileName -> filter == null || LSMTimeline.isFileInRange(filter, fileName))
           .parallel().forEach(fileName -> {
             // Read the archived file
-            try (HoodieAvroParquetReader reader = (HoodieAvroParquetReader) HoodieFileReaderFactory.getReaderFactory(HoodieRecordType.AVRO)
+            try (HoodieAvroFileReaderBase reader = (HoodieAvroFileReaderBase) HoodieFileReaderFactory.getReaderFactory(HoodieRecordType.AVRO)
```

Comment:
A side topic, what is the difference between `HoodieAvroFileReader` and `HoodieAvroFileReaderBase`?  I think we can rename `HoodieAvroFileReaderBase` to `HoodieAvroFileReader` if there is no difference and remove the interface `HoodieAvroFileReader`.

Reply:
I don't see any reason why not to do that